### PR TITLE
[CI-1255] Migrate to tool kit

### PR DESCRIPTION
**Pull request by AlbertoCuberoNavas**
[https://financialtimes.atlassian.net/browse/CI-1255](https://financialtimes.atlassian.net/browse/CI-1255)

### CODE REVIEW

**Review by juanSanchezAlcala — CHANGES_REQUESTED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,220 +1,82 @@
...
+            tags:
+              only: /^v\d+\.\d+\.\d+(-.+)?/
+      - tool-kit/build:
+          node-version: 16.14-browsers
+          requires:
+            - tool-kit/setup
           filters:
-            <<: *filters_version_tag
+            tags:
+              only: /^v\d+\.\d+\.\d+(-.+)?/
```
[the remark this answers is not in the document]
Comment (AlbertoCuberoNavas) [reply .circleci/config.yml:46]: its auto-added when you run `npx dotcom-tool-kit --install`. If you remove this line and then you modify .toolkitrc.yml and run `npx dotcom-tool-kit --install` again, this line will be added.

This is a toolkit replication based on n-exponea

File: .circleci/config.yml [hunk 2]
```diff
@@ -1,220 +1,82 @@
...
-          matrix:
-            parameters:
-              node-version: [ "16.14", "14.19" ]
-      - publish:
-          context: npm-publish-token
+            - tool-kit/build
           filters:
-            <<: *filters_version_tag
+            tags:
+              only: /^v\d+\.\d+\.\d+(-.+)?/
```
[the remark this answers is not in the document]
Comment (AlbertoCuberoNavas) [reply .circleci/config.yml:53]: its auto-added when you run npx dotcom-tool-kit --install. If you remove this line and then you modify .toolkitrc.yml and run npx dotcom-tool-kit --install again, this line will be added.

This is a toolkit replication based on n-exponea

File: .circleci/config.yml [hunk 3]
```diff
@@ -1,220 +1,82 @@
...
-      - test:
-          name : test-no-main-v<< matrix.node-version >>
-          requires:
-            - build-no-main-v<< matrix.node-version >>
-          matrix:
-            parameters:
-              node-version: [ "16.14", "14.19" ]
-      - wait-for-approval:
+            tags:
+              only: /^v\d+\.\d+\.\d+(-.+)?/
```
[the remark this answers is not in the document]
Comment (AlbertoCuberoNavas) [reply .circleci/config.yml:25]: its auto-added when you run npx dotcom-tool-kit --install. If you remove this line and then you modify .toolkitrc.yml and run npx dotcom-tool-kit --install again, this line will be added.

This is a toolkit replication based on n-exponea

File: .circleci/config.yml [hunk 4]
```diff
@@ -1,220 +1,82 @@
...
-            - wait-for-approval
-          matrix:
-            parameters:
-              node-version: [ "16.14", "14.19" ]
-      - test:
-          name : test-renovate-v<< matrix.node-version >>
+            branches:
+              only: /(^renovate-.*|^nori/.*)/
+            tags:
+              only: /^v\d+\.\d+\.\d+(-.+)?/
```
[the remark this answers is not in the document]
Comment (AlbertoCuberoNavas) [reply .circleci/config.yml:32]: this is for renovate and nori branches only

**Review by juanSanchezAlcala — APPROVED** [review]

**Pull request merged**